### ISSUE TITLE
Include a 4.12 compatible menuset.

### ISSUE THREAD

**Issue by wwiv** [wwivbbs/wwiv#1321]
This will also need some custom prompt work to make this happen.  We can also tweak the yourinfo page (since that can be handled by a MSG file now) as well.

**Comment by wwiv** [comment]
WWIV 4.12 had different style *.ANS/*.MSG menus (used thicker blocks).  So it'd be nice to have one (just using .MSG with pipe codes, no ANS needed anymore) that used those menus for main and transfer.  Also should try to make the bbslist, automsg look consistent with the yellow prompt like "R:ead" vs. (R)ead as we use now.

There also may be support needed in the bbs to help customize this, so this could be a good way to find out what's missing customization wise.

Also some features (like (Y)ourinfo), could likely be redone as a text file using PIPE macro's vs. the bbs's ``yourinfo``` menu command (actually that may generally be the way to go for this command now)

**Comment by wwiv** [comment]
(Y)ourinfo is now done with a textfile, so that's ready.
Next up is we'll need some customizable prompt support.

**Comment by ericpareja** [comment]
Do you mean these? 4.11 and 4.20 have the same menu look.

![Screenshot from 2023-01-17 23-42-53](https://user-images.githubusercontent.com/188277/212943442-1a07d9f8-365c-4fcc-a7a5-1e55f8936968.png)

**Comment by ericpareja** [comment]
These are from 4.10
![Screenshot from 2023-01-17 23-44-35](https://user-images.githubusercontent.com/188277/212943958-d3fda21c-aa80-4dd2-a4d5-2c942a90a06c.png)

**Comment by ericpareja** [comment]
This is from 4.06
![Screenshot from 2023-01-17 23-46-57](https://user-images.githubusercontent.com/188277/212944588-f96d2b3f-c4fa-4d86-a96a-544ebd977d09.png)

**Comment by ericpareja** [comment]
This is from v4.22
![Screenshot from 2023-01-18 00-08-17](https://user-images.githubusercontent.com/188277/212951311-698ab824-3f3f-4716-82be-0e4f6449335c.png)

**Comment by ericpareja** [comment]
From v4.24
![Screenshot from 2023-01-18 00-09-31](https://user-images.githubusercontent.com/188277/212951556-ba7d0195-582f-49ef-9d43-c04884bd0c0c.png)

**Comment by wwiv** [comment]
yes and also prompts that match for things like bbs list, automessage, etc
